supernovacontroller/sequential/i3c.py

```diff
@@ -808,20 +816,32 @@ def ccc_getmwl(self, target_address):
 
     def ccc_getxtime(self, target_address):
         """
-        Performs a GETXTIME (Get Extra Timing Information) operation on a target device on the I3C bus.
+        Performs a GETXTIME (Get Exchange Timing Information) operation on a target device on the I3C bus.
 
-        This method requests the Extra Timing Information from the specified target device.
+        This method requests the Exchange Timing Information from the specified target device.
         The operation's success status is checked, and it returns a tuple indicating whether the operation
         was successful along with the relevant data or error message.
 
         Args:
-        target_address: The address of the target device on the I3C bus from which the Extra Timing Information is requested.
+        target_address: The address of the target device on the I3C bus from which the Exchange Timing Information is requested.
 
         Returns:
         tuple: A tuple containing two elements:
             - The first element is a Boolean indicating the success (True) or failure (False) of the operation.
-            - The second element is either a dictionary containing the Extra Timing Information and its length, indicating
-                success, or an error message detailing the failure.
+            - The second element is either a dictionary containing the Exchange Timing Information and its length, indicating
+                success, or an error message detailing the failure. The dictionary is of shape:
+                ```
+                {
+                    "supportedModesByte": int,
+                    "stateByte": int,
+                    "frequency": int
+                    "inaccuracy": int
```

Comment:
According to the documentation of the mipi spec v1.1.1, in the section 5.1.9.3.22 it states that all data pieces are "bytes", i.e., we have the Supported Modes Byte, the State Byte, the Frequency Byte and the Inaccuracy Byte. Why are we adding the word "byte" only to 2 of the 4 data elements?

Reply:
Whilst they are all bytes (all computer code is), the difference is how those are used.
SupportedMode and State are marked as bytes since their numerical value means nothing (kinda), its bit by bit meaning is what matters (kind of like flags)
Frequency and inaccuracy are both values to be taken as is, numerical.